main.tex

```diff
@@ -379,12 +386,12 @@ \section{Results}
 Figure~\ref{fig:results} shows the results of our experiments. In this section, we discuss the results and how they relate to the three research questions outlined in the previous section.
 
 \subsection{Multiprocessing}
-With batch size 1, the testing speed scales almost linearly with the number of \acro{CPU}s, as we would expect. Whereas with with 1~\acro{CPU}, the median testing time is 27~hours and 2~minutes, it is only 1~hour and 6~minutes with 32~\acro{CPU}s (about 24-fold speed-up).\footnote{%
-The reason why we did not achieve the theoretical 32-fold speedup is likely due to tasks from other users on our server.%
+With batch size 1, the testing speed scales almost linearly with the number of \acro{CPU}s, as we would expect: with 1~\acro{CPU}, the median testing time is 27~hours and 2~minutes, it is only 1~hour and 6~minutes with 32~\acro{CPU}s (about 24-fold speed-up).\footnote{%
```

Comment:
I assume that you left out "Whereas" here. Without it, the "it is only ..." part of the last clause seems misplaced.

```suggestion
With batch size 1, the testing speed scales almost linearly with the number of \acro{CPU}s, as we would expect: Whereas with 1~\acro{CPU}, the median testing time is 27~hours and 2~minutes, it is only 1~hour and 6~minutes with 32~\acro{CPU}s (about 24-fold speed-up).\footnote{%
```